### PR TITLE
Update signed-apk-android.md

### PR DESCRIPTION
**Pull request by JavaJeremy**
The description didn't work for me, after lots of googling and trying out I finally found out, it worked for me in the jre folder and with the command with "./" at the beginning (<a href="https://stackoverflow.com/questions/16333635/keytool-error-bash-keytool-command-not-found?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa">SO /.</a>) AND only in admin mode (<a href="https://stackoverflow.com/questions/10321211/java-keytool-error-after-importing-certificate-keytool-error-java-io-filenot?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa">SO Admin</a>)

Thank you for the PR! Contributors like you keep React Native awesome!

Please see the Contribution Guide for guidelines:

https://github.com/facebook/react-native-website/blob/master/CONTRIBUTING.md

If your PR references an existing issue, please add the issue number below:

#<Issue>

### CODE REVIEW

File: docs/signed-apk-android.md [hunk 1]
```diff
@@ -7,9 +7,9 @@ Android requires that all apps be digitally signed with a certificate before the
 
 ### Generating a signing key
 
-You can generate a private signing key using `keytool`. On Windows `keytool` must be run from `C:\Program Files\Java\jdkx.x.x_x\bin`.
+You can generate a private signing key using `keytool`. On Windows `keytool` must be run from `C:\Program Files\Java\jdkx.x.x_x\bin` or `C:\Program Files\Java\jrex.x.x_x\bin` in administrator mode.
 
-    $ keytool -genkey -v -keystore my-release-key.keystore -alias my-key-alias -keyalg RSA -keysize 2048 -validity 10000
+    $ ./keytool -genkey -v -keystore my-release-key.keystore -alias my-key-alias -keyalg RSA -keysize 2048 -validity 10000
```
Comment (charpeni) [inline docs/signed-apk-android.md]: `./keytool` won't work on Mac or Linux.

Can you reword this as Windows specificity in the paragraph below?

File: docs/signed-apk-android.md [hunk 2]
```diff
@@ -7,9 +7,9 @@ Android requires that all apps be digitally signed with a certificate before the
 
 ### Generating a signing key
 
-You can generate a private signing key using `keytool`. On Windows `keytool` must be run from `C:\Program Files\Java\jdkx.x.x_x\bin`.
+You can generate a private signing key using `keytool`. On Windows `keytool` must be run from `C:\Program Files\Java\jdkx.x.x_x\bin` or `C:\Program Files\Java\jrex.x.x_x\bin` in administrator mode.
```
[the remark this answers is not in the document]
Comment (JavaJeremy) [reply docs/signed-apk-android.md]: I think you might be right, but I do not have much knowledge about this stuff -
I don't know how this example would look like - feel free to request a change to this commit or something like that

**Comment by cpojer** [comment]
Closing because this PR has been stale for a long time.
